src/sentry/integrations/slack/threads/activity_notifications.py

```diff
@@ -0,0 +1,11 @@
+from collections.abc import Mapping
+from typing import Any
+
+from sentry.notifications.notifications.activity.assigned import (
+    AssignedActivityNotification as BaseAssignedActivityNotification,
+)
+
+
+class AssignedActivityNotification(BaseAssignedActivityNotification):
```

Comment:
should we call this class something else if people get confused about which one to use?

Reply:
I think we the base/original ones should be renamed to use `email/html` as the module name instead of the general "notification" name it has. What do you think?

Reply:
activity notifications can also send slack DMs :(

<img width="667" alt="Screenshot 2024-05-01 at 13 33 49" src="https://github.com/getsentry/sentry/assets/70817427/d26aba45-c456-4bd5-9b36-c8cdcb061337">

Reply:
I think the module names should be read and considered when importing. To help delineate the difference, let me add doc strings to help describe why this exists

Reply:
done, added some comments